plugins/linux-v4l2/v4l2-output.c

```diff
@@ -75,8 +75,7 @@ static bool loopback_module_loaded()
 		}
 	}
 
-	if (fp)
-		fclose(fp);
+	fclose(fp);
```

Comment:
When `fp` is NULL, `fclose` should not be called.
```suggestion
	if (fp)
		fclose(fp);
```